### DISCUSSION TITLE
[projects beta] Increase the size of the status field

### DISCUSSION THREAD

**Discussion by NotDopplerDing** [community/community#38444] [Projects and Issues]
Hi,

Currently the size of the status field is limit to 50, I wonder if it can be increased as we are using it to categorise sprint.

We can close the old one, but it would good if we can keep this as a history of our development.

Thanks

**Comment by ossanna16** [comment]
Hi there @NotDopplerDing and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by NotDopplerDing** [reply]
Thank you for your reply. I am looking forward to a developer respond.
